neo.UnitTests/UT_RpcClient.cs

```diff
@@ -0,0 +1,540 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Moq.Protected;
+using Neo.IO;
+using Neo.IO.Json;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Network.RPC;
+using Neo.Network.RPC.Models;
+using Neo.SmartContract;
+using Neo.SmartContract.Manifest;
+using Neo.VM;
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_RpcClient
```

Comment:
Should this file be moved into a sub folder?

Reply:
Yes, I have moved the file to Network/RPC folder